src/Symfony/Bridge/PhpUnit/Extension/UnregisterClockMockSubscriber.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Extension;
+
+use PHPUnit\Event\Test\Finished;
+use PHPUnit\Event\Test\FinishedSubscriber;
+use PHPUnit\Metadata\Group;
+use Symfony\Bridge\PhpUnit\ClockMock;
+
+class UnregisterClockMockSubscriber implements FinishedSubscriber
```

Comment:
this subscriber disables the mock, but it does *not* unregister it (which is not possible anyway)